luminance/src/pixel.rs

```diff
@@ -880,6 +884,20 @@ impl_Pixel!(
 impl_ColorPixel!(R11G11B10F);
 impl_RenderablePixel!(R11G11B10F);
 
+/// A blue, green, red and alpha 8-bit unsigned integral pixel format.
+#[derive(Clone, Copy, Debug)]
+pub struct NormBGRA8UI;
+
+impl_Pixel!(
+  NormBGRA8UI,
+  (u8, u8, u8, u8),
+  u8,
+  NormUnsigned,
+  Format::BGRA(Size::Eight, Size::Eight, Size::Eight, Size::Eight)
+);
+impl_ColorPixel!(NormBGRA8UI);
+impl_RenderablePixel!(NormBGRA8UI);
```

Comment:
Unfortunately, it’s not renderable. 🙁 

Source:
> [An internal format iscolor-renderableif it isRED,RG,RGB,RGBA](https://www.khronos.org/registry/OpenGL/specs/gl/glspec46.core.pdf)

Reply:
That explains it ;(